### ISSUE TITLE
Feature request: input jump in logs

### ISSUE THREAD

**Issue by XLTechie** [CyrilleB79/NVDA-Dev-Test-Toolbox#11]
I know your log viewer enhancements currently provide the ability to jump between IO events. While that is useful, may I suggest adding a feature to jump to I events only?

I often find it necessary to look for a specific input line, to start debugging something. The I command will be useful for this, but since it also catches on speak events, it would be even better if there was a way to jump through input events only.

**Comment by CyrilleB79** [comment]
Yes, having a more refined filtering when navigating through IO entries makes sense. Actually, the issue become much more  annoying when you are using a braille display.

**Comment by XLTechie** [comment]
Perhaps alter the commands, using i for input, and o for output (speak/braille)?

Assuming you can jump at that level of granularity.

**Comment by CyrilleB79** [comment]
@XLTechie in commit aeaecc5bf0329561378b24f5f347945e84fde331, I have added the possibility to jump to logged inputs (key n) and speech (key s). To me these are the most important IO messages for debug.

I do not usually debug braille; but probably something would be useful for such debug.

New types of message can easily be added in the future, but I prefer to wait for the need to arise.

These changes, as well as markers in the log (thanks !), will be available in the next release.

Is it OK for you or do you think that I have missed some important jump commands?

**Comment by XLTechie** [comment]
> @XLTechie in commit aeaecc5, I have added the possibility to jump to logged inputs (key n) and speech (key s). To me these are the most important IO messages for debug.

Agreed, and thanks! That will be very helpful to me.

**Comment by CyrilleB79** [comment]
Included in V5.0.
